src/main.c

```diff
@@ -302,7 +286,24 @@ main (int argc, char *argv[])
                               time_now, time_str_appended, current_waste_num);
 
             if (info_status == 0)
+            {
```

Comment:
Ok, no worries.

I'm on Quakenet IRC in #rmw you want to stay in touch through this process.